hotel-api-services/src/main/java/com/fponce/hotelapp/hotel/HotelController.java

```diff
@@ -26,9 +28,16 @@
     }
 
     @PostMapping(consumes = {MediaType.APPLICATION_JSON_VALUE})
-    public ResponseEntity createHotel(@Valid @RequestBody Hotel hotel) throws HotelAppServicesException {
+    public ResponseEntity createHotel(@RequestBody Hotel hotel) throws HotelAppServicesException {
```

Comment:
what do you think about returning a `Resource<>` instead of a `ResponseEntity`
as we are working in a REST API, I consider it would be a little more helpful
more info: https://docs.spring.io/spring-hateoas/docs/current/api/org/springframework/hateoas/Resource.html

Reply:
Do you mean I should return the created entity as part of the Resource? 
``return new Resource(createdHotel);
I found this but I don´t know if it is valid, since it is not an official documentation
https://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#useful-post-responses
"Updates & creation should return a resource representation"

Reply:
Yes, check this link, this would be useful: https://github.com/spring-guides/tut-rest